ocaml-ci-web.opam

```diff
@@ -40,3 +37,10 @@ build: [
   ]
 ]
 dev-repo: "git+https://github.com/ocurrent/ocaml-ci.git"
+
+pin-depends: [
+  ["dream.dev" "git+https://github.com/novemberkilo/dream#4976439384c9da03300abdbc9ffb8528ac917982"]
+  ["dream-httpaf.dev" "git+https://github.com/novemberkilo/dream#4976439384c9da03300abdbc9ffb8528ac917982"]
+  ["dream-mirage.dev" "git+https://github.com/novemberkilo/dream#4976439384c9da03300abdbc9ffb8528ac917982"]
```

Comment:
Do you need `dream-mirage.dev` here? I wouldn't expect we install that in the opam switch.